README.zh-CN.md

```diff
@@ -61,27 +61,569 @@ Apache EventMesh提供了许多功能来帮助用户实现他们的目标，以
 - [EventMesh-catalog](https://github.com/apache/eventmesh-catalog): 使用 AsyncAPI 进行事件模式管理的目录服务。
 - [EventMesh-go](https://github.com/apache/eventmesh-go): EventMesh 运行时的 Go 语言实现。
 
-## 快速开始
+## 快速入门  
+本段指南将指导您完成EventMesh的部署步骤   
+- [部署EventMesh Store](#部署eventmesh-store-)
```

Comment:
There should be a blank line between the title and the paragraph in markdown. Here, you need to add a blank line below the title and above the unordered list to avoid some stricter markdown renderers displaying the two paragraphs on the same line.